JSDemos/Demos/TabPanel/Overview/Vue/App.vue

```diff
@@ -1,136 +1,190 @@
-              (<span>{{ company.State }}</span>)
-            </p>
-            <p>
-              <span>{{ company.Zipcode }}&nbsp;</span>
-              <span>{{ company.Address }}</span>
-            </p>
-          </div>
-          <div>
-            <p>
-              Phone: <b>{{ company.Phone }}</b>
-            </p>
-            <p>
-              Fax: <b>{{ company.Fax }}</b>
-            </p>
-            <p>
-              Website:
-              <a
-                :href="company.Website"
-                target="_blank"
-              >
-                {{ company.Website }}
-              </a>
-            </p>
-          </div>
-        </div>
-      </template>
-    </DxTabPanel>
-    <div class="item-box">
-      Item <span>{{ selectedIndex + 1 }}</span> of <span>{{ companies.length }}</span>
+  <div class="tabpanel-demo">
+    <div class="widget-container">
+      <DxTabPanel
+        class="dx-theme-background-color"
+        width="100%"
+        height="418"
+        item-template="tabPanelItem"
+        :animation-enabled="true"
+        :swipe-enabled="true"
+        :data-source="dataSource"
+        :selected-index="selectedIndex"
+        :tabs-position="tabsPosition"
+        :styling-mode="stylingMode"
+        :icon-position="iconPosition"
+        :show-nav-buttons="showNavButtons"
+        :on-option-changed="onSelectionChanged"
+      >
+        <template #tabPanelItem="{ data }">
+          <TabPanelItem
+            :tasks="data.tasks"
+          />
+        </template>
+      </DxTabPanel>
     </div>
+
     <div class="options">
       <div class="caption">Options</div>
+
       <div class="option">
-        <DxCheckBox
-          v-model:value="loop"
-          text="Loop enabled"
+        <div class="option-label">Tabs position</div>
+
+        <DxSelectBox
+          :input-attr="tabsPositionsSelectBoxLabel"
+          :items="tabsPositions"
+          :value="tabsPosition"
+          :on-value-changed="onTabsPositionChanged"
         />
       </div>
+
       <div class="option">
-        <DxCheckBox
-          v-model:value="animationEnabled"
-          text="Animation enabled"
+        <div class="option-label">Styling mode</div>
+
+        <DxSelectBox
+          :input-attr="stylingModesSelectBoxLabel"
+          :items="stylingModes"
+          :value="stylingMode"
+          :on-value-changed="onStylingModeChanged"
+        />
+      </div>
+
+      <div class="option">
+        <div class="option-label">Icon position</div>
+
+        <DxSelectBox
+          :input-attr="iconPositionsSelectBoxLabel"
+          :items="iconPositions"
+          :value="iconPosition"
+          :on-value-changed="onIconPositionChanged"
         />
       </div>
+
       <div class="option">
         <DxCheckBox
-          v-model:value="swipeEnabled"
-          text="Swipe enabled"
+          text="Show navigation buttons"
+          :element-attr="navButtonsCheckBoxLabel"
+          :value="showNavButtons"
```

Comment:
Can we use two way binding here and other places via v-model:value="showNavButtons"?

Reply:
[9314317e2a169f6a48660ae0ab31835edd5370e8]